mkdocs.yml

```diff
@@ -541,49 +541,39 @@ nav:
-        - 'Upgrading to eZ Platform v3': 'upgrading/upgrading_to_v3.md'
-        - '1. Check out a tagged version': 'upgrading/1_check_out_version.md'
-        - '2. Merge composer.json': 'upgrading/2_merge_composer.md'
-        - '3. Update the app': 'upgrading/3_upgrade_app.md'
-        - '4. Upgrade the code':
-            - '4. Upgrade the code': 'upgrading/4_upgrade_the_code.md'
-            - '4.1. Upgrade templates': 'upgrading/4_1_upgrade_templates.md'
-            - '4.2. Upgrade configuration': 'upgrading/4_2_upgrade_configuration.md'
-            - '4.3. Upgrade Field Types': 'upgrading/4_3_upgrade_field_types.md'
-            - '4.4. Upgrade Signal Slots': 'upgrading/4_4_upgrade_signal_slots.md'
-            - '4.5. Upgrade Online Editor': 'upgrading/4_5_upgrade_online_editor.md'
-            - '4.6. Upgrade workflow': 'upgrading/4_6_upgrade_workflow.md'
-            - '4.7. Upgrade extended code': 'upgrading/4_7_upgrade_extensions.md'
-            - '4.8. Upgrade REST': 'upgrading/4_8_upgrade_rest.md'
-            - '4.9 Other code upgrades': 'upgrading/4_9_upgrade_other.md'
-        - '5. Upgrade the database': 'upgrading/5_upgrade_database.md'
-        - '6. Platform.sh changes': 'upgrading/6_platform_sh_changes.md'
-        - '7. Dump assets': 'upgrading/7_dump_assets.md'
-        - '8. Commit, test and merge': 'upgrading/8_commit_test_merge.md'
-    - Updating Ibexa DXP:
-        - 'Updating to v3.3': 'updating/updating_to_3.3.md'
-        - 'Updating Ibexa DXP': 'updating/updating_ez_platform.md'
-        - '1. Check out a tagged version': 'updating/1_check_out_version.md'
-        - '2. Merge composer.json': 'updating/2_merge_composer.md'
-        - '3. Update the app': 'updating/3_update_app.md'
-        - '4. Update database':
-          - 'Update database': 'updating/4_update_database.md'
-          - 'Updating from <1.7': 'updating/4_update_1.7.md'
-          - 'Updating from <1.13': 'updating/4_update_1.13.md'
-          - 'Updating from <2.2': 'updating/4_update_2.2.md'
-          - 'Updating from <2.3': 'updating/4_update_2.3.md'
-          - 'Updating from <2.4': 'updating/4_update_2.4.md'
-          - 'Updating from <2.5': 'updating/4_update_2.5.md'
-          - 'Updating from <3.1': 'updating/4_update_3.1.md'
-          - 'Updating from <3.2': 'updating/4_update_3.2.md'
-        - '5. Platform.sh changes': 'updating/5_platform_sh_changes.md'
-        - '6. Dump assets': 'updating/6_dump_assets.md'
-        - '7. Commit, test and merge': 'updating/7_commit_test_merge.md'
+    - Update and migration:
+        - Updating Ibexa DXP:
+            - 'Updating to v3.3': 'updating/updating_to_3.3.md'
+            - 'Updating Ibexa DXP': 'updating/updating.md'
+            - '1. Check out a version': 'updating/1_check_out_version.md'
+            - '2. Resolve conflicts': 'updating/2_merge_composer.md'
+            - '3. Update the app': 'updating/3_update_app.md'
+            - '4. Upgrade the code':
```

Comment:
Is there a reason why some things are updated` (app, database) and some are upgraded (code)?

Reply:
The idea was that update is smaller and more automatic, while upgrade is larger, requiring more hands-on work (as with upgrading custom code to v3), but I'm not married to the idea, let's see what the rest of the doc team has to say about it.